### DISCUSSION TITLE
Welcome to Cauldron Discussions!

### DISCUSSION THREAD

**Discussion by TimRohr22** [TimRohr22/Cauldron#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amcolemanacz** [comment]
Quick question, on name If the player's name shows up in Roll20 as 'John S.' is the full name including period required or would just '--John' work?
